style.css

```diff
@@ -33,6 +40,23 @@ div.navbar {
    align: center;
 }
 
+#desc {
```

Comment:
Use classes instead of IDs to refer in your CSS. It's ok to call it description instead of desc, that makes it more readable